tests/test_platform_maps.py

```diff
@@ -0,0 +1,492 @@
+"""
+Unit tests for platform_maps module.
+
+Tests verify:
+- JSON loading from platform_maps.json
+- Backward compatibility with existing code
+- Fallback behavior when JSON is missing/invalid
+- CUSTOM_MAPS environment variable override
+- Module exports and interfaces
+"""
```

Comment:
waaaay too many tests for like 30 lines of changes lol, strip this down to like 2-3 tests please